apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/CreateProfile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -4,8 +4,8 @@ import { type CreateProfileFormValues } from '@/app/[locale]/(profile)/profile.t
 import InputFormError from '@/components/InputFormError/InputFormError'
```

Comment:
**nitpick**: Keep import statements ordered by external and then internal modules for better readability.